frontend/src/components/SecureExchange/AccessUsersPage.vue

```diff
@@ -72,6 +72,7 @@ export default {
   },
   created() {
     this.getUsersData();
+    console.log('USER INFO: ' + JSON.stringify(this.userInfo, null, 3));
```

Comment:
Please remove

Reply:
Same with this.... am I losing it? https://github.com/bcgov/EDUC-EDX/blob/feature/edx-10/frontend/src/components/SecureExchange/AccessUsersPage.vue